src/replica/storage/simple_kv/test/case-207.act

```diff
@@ -64,77 +64,66 @@ client:replica_config:receiver=r1,type=downgrade_to_inactive,node=r1
-client:end_read:id=4,err=err_ok,resp=<<not-exist>>
+client:end_read:id=4,err=err_ok,resp=v4
 client:begin_read:id=5,key=k5,timeout=0
-client:end_read:id=5,err=err_ok,resp=<<not-exist>>
+client:end_read:id=5,err=err_ok,resp=v5
 client:begin_read:id=6,key=k6,timeout=0
-client:end_read:id=6,err=err_ok,resp=<<not-exist>>
+client:end_read:id=6,err=err_ok,resp=v6
 client:begin_read:id=7,key=k7,timeout=0
-client:end_read:id=7,err=err_ok,resp=<<not-exist>>
+client:end_read:id=7,err=err_ok,resp=v7
 client:begin_read:id=8,key=k8,timeout=0
-client:end_read:id=8,err=err_ok,resp=<<not-exist>>
+client:end_read:id=8,err=err_ok,resp=v8
 client:begin_read:id=9,key=k9,timeout=0
-client:end_read:id=9,err=err_ok,resp=<<not-exist>>
+client:end_read:id=9,err=err_ok,resp=v9
 client:begin_read:id=10,key=k10,timeout=0
-client:end_read:id=10,err=err_ok,resp=<<not-exist>>
+client:end_read:id=10,err=err_ok,resp=v10
 
-# change primary from r3 to r2
-client:replica_config:receiver=r3,type=downgrade_to_secondary,node=r3
-config:{7,-,[r2,r3]}
-state:{{r2,sec,7,0},{r3,sec,7,0}}
+# add secondary r2
+client:replica_config:receiver=r1,type=add_secondary,node=r2
+config:{6,r1,[r2]}
+state:{{r2,sec,6,10},{r1,pri,6,10}}
+
+# change primary from r1 to r2
+client:replica_config:receiver=r1,type=downgrade_to_secondary,node=r1
+config:{7,-,[r2,r1]}
+state:{{r2,sec,7,10},{r1,sec,7,10}}
 client:replica_config:receiver=r2,type=upgrade_to_primary,node=r2
-config:{8,r2,[r3]}
-state:{{r2,pri,8,0},{r3,sec,8,0}}
+config:{8,r2,[r1]}
+state:{{r2,pri,8,10},{r1,sec,8,10}}
 
 # check r2 data corrent
 client:begin_read:id=1,key=k1,timeout=0
 wait:on_rpc_call:rpc_name=RPC_SIMPLE_KV_SIMPLE_KV_READ,from=c,to=r2
-client:end_read:id=1,err=err_ok,resp=<<not-exist>>
+client:end_read:id=1,err=err_ok,resp=v1
 client:begin_read:id=2,key=k2,timeout=0
-client:end_read:id=2,err=err_ok,resp=<<not-exist>>
+client:end_read:id=2,err=err_ok,resp=v2
 client:begin_read:id=3,key=k3,timeout=0
-client:end_read:id=3,err=err_ok,resp=<<not-exist>>
+client:end_read:id=3,err=err_ok,resp=v3
 client:begin_read:id=4,key=k4,timeout=0
-client:end_read:id=4,err=err_ok,resp=<<not-exist>>
+client:end_read:id=4,err=err_ok,resp=v4
 client:begin_read:id=5,key=k5,timeout=0
-client:end_read:id=5,err=err_ok,resp=<<not-exist>>
+client:end_read:id=5,err=err_ok,resp=v5
 client:begin_read:id=6,key=k6,timeout=0
-client:end_read:id=6,err=err_ok,resp=<<not-exist>>
+client:end_read:id=6,err=err_ok,resp=v6
 client:begin_read:id=7,key=k7,timeout=0
-client:end_read:id=7,err=err_ok,resp=<<not-exist>>
+client:end_read:id=7,err=err_ok,resp=v7
 client:begin_read:id=8,key=k8,timeout=0
-client:end_read:id=8,err=err_ok,resp=<<not-exist>>
+client:end_read:id=8,err=err_ok,resp=v8
 client:begin_read:id=9,key=k9,timeout=0
-client:end_read:id=9,err=err_ok,resp=<<not-exist>>
+client:end_read:id=9,err=err_ok,resp=v9
 client:begin_read:id=10,key=k10,timeout=0
-client:end_read:id=10,err=err_ok,resp=<<not-exist>>
-
+client:end_read:id=10,err=err_ok,resp=v10
```

Comment:
Old case207 will test lost data, new primary3 won't have data [1, 10]. In my opinion, this case should remain the lost-data logic. 
How about updating this case like following?
```
// just smiliar like case208 and case209, write some data

=> state {{r1,pri,2,10},{r2,sec,2,10}} 
remove sec r2 
=>state{{r1,pri,2,10}} 
add secondary r3 (new replica) 
=>state:{{r1,pri,4,10},{r3,sec,4,10}} 
write data #11-#15 
=>state:{{r1,pri,4,15,15},{r3,sec,4,15,15}} 
remove sec r3 and pri r1 
=>state:{}
active pri r2 and check data 
state:{{r2,pri,7,10,10}} 

// The following not contain the state, just pseudocode
add sec r3 and check data (#1-#10 correct, #11-#15 not-exist)
change pri to r3, sec to r2 and check data state (#1-#10 correct, #11-#15 not-exist)
```